ui/apps/everest/.e2e/release/init-deploy.e2e.ts

```diff
@@ -0,0 +1,180 @@
+// everest
+// Copyright (C) 2023 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { expect, test } from '@playwright/test';
+import {
+  deleteDbCluster,
+  suspendDbCluster,
+  resumeDbCluster,
+  restartDbCluster,
+} from '../utils/db-clusters-list';
+import { getTokenFromLocalStorage } from '../utils/localStorage';
+import { getClusterDetailedInfo } from '../utils/storage-class';
+import { resourcesStepCheck } from '../db-cluster/db-wizard/create-db-cluster/steps/resources-step';
+import {
+  moveForward,
+  submitWizard,
+  populateBasicInformation,
+} from '../utils/db-wizard';
+import { EVEREST_CI_NAMESPACES } from '../constants';
+import { waitForStatus, waitForDelete } from '../utils/table';
+
+test.describe.configure({ retries: 0 });
+
+[
+  { db: 'psmdb', size: 1 },
+  { db: 'psmdb', size: 3 },
+  { db: 'pxc', size: 1 },
+  { db: 'pxc', size: 3 },
+  { db: 'postgresql', size: 1 },
+  { db: 'postgresql', size: 3 },
+].forEach(({ db, size }) => {
+  test.describe(
+    'Init deploy workflow',
+    {
+      // Create/stop/resume/delete cluster in different configurations
+      tag: '@release',
+    },
+    () => {
+      const clusterName = `${db}-${size}-deploy`;
+
+      let storageClasses = [];
+      let namespace = EVEREST_CI_NAMESPACES.EVEREST_UI;
+
+      test.beforeAll(async ({ request }) => {
+        const token = await getTokenFromLocalStorage();
+
+        const { storageClassNames = [] } = await getClusterDetailedInfo(
+          token,
+          request
+        );
+        storageClasses = storageClassNames;
+      });
+
+      test(`Cluster creation with ${db} and size ${size}`, async ({
+        page,
+        request,
+      }) => {
+        test.setTimeout(size * 120000);
+
+        expect(storageClasses.length).toBeGreaterThan(0);
+
+        await page.goto('/databases/new');
```

Comment:
Any specific reason you would like to keep URL's and locators for pages directly in the test files, could they be part of page classes or some other helpers?

Reply:
No specific reason, I was looking at page object pattern as well, but current setup is not like that so didn't want to change to much. But it can be done.